### DISCUSSION TITLE
Single card component not rendering

### DISCUSSION THREAD

**Discussion by tsekeres** [nss-evening-cohort-14/evening-client-side#294] [Capstones!] [answered]
Tad S
Describe your issue:
Not getting SingleTripCard to render. Info and props seem to be getting all the way to the View but then cannot get through to the last Component.

List any error messages that appear:
No errors

What have you googled?
https://stackoverflow.com/questions/30465651/passing-keys-to-children-in-react-js
https://stackoverflow.com/questions/41934077/pass-key-param-to-react-component

What have you tried?
I've tried may iterations of the key.
Tried many different props passed to SingleTripCard component.

Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with:
https://github.com/tsekeres/appy-trails-capstone/blob/setup/src/views/SingleTrip.js

**Comment by TrinityChristiana** [comment]
@tsekeres Will add their solution.

**Comment by tsekeres** [comment, marked as answer]
I had the wrong folder/node name in my promise. But the debugging process was very helpful to watch. Thanks Trinity.
